src/Modules/Dashboard/Dashboard.tsx

```diff
@@ -1,7 +1,16 @@
-import React from 'react'
+
+import { useNavigate } from 'react-router-dom';
 
 export default function Dashboard() {
+
+  let navigate = useNavigate();
+  let handleClick=()=>{
+    localStorage.removeItem('token');
```

Comment:
We need to clear the whole local storage not just the token. The localStorage has a clear method.

Reply:
done,
localStorage.clear()